go-controller/pkg/ovn/obj_retry.go

```diff
@@ -0,0 +1,138 @@
+package ovn
+
+import (
+	"time"
+
+	"k8s.io/klog/v2"
+)
+
+const retryObjInterval = 30 * time.Second
+
+type retryObjEntry struct {
+	newObj     interface{}
+	oldObj     interface{}
+	config     interface{}
```

Comment:
so this is a generic config object that resource specfic handlers may use to store internal information. Maybe we should add a comment here indicating that.